boards/px4/fmu-v5x/init/rc.board_defaults

```diff
@@ -3,23 +3,25 @@
 # board specific defaults
 #------------------------------------------------------------------------------
 
-# Mavlink ethernet (CFG 1000)
-param set-default MAV_2_CONFIG 1000
-param set-default MAV_2_BROADCAST 1
-param set-default MAV_2_MODE 0
-param set-default MAV_2_RADIO_CTL 0
-param set-default MAV_2_RATE 100000
-param set-default MAV_2_REMOTE_PRT 14550
-param set-default MAV_2_UDP_PRT 14550
-
 param set-default SENS_EN_INA238 0
 param set-default SENS_EN_INA228 0
 param set-default SENS_EN_INA226 1
 
 if ver hwtypecmp V5X009000 V5X009001 V5X00a000 V5X00a001 V5X008000 V5X008001 V5X010001
 then
-	# Skynode: use the "custom participant" config for uxrce_dds_client
+	# Skynode: use the "custom participant", IP=10.41.10.1 config for uxrce_dds_client
 	param set-default UXRCE_DDS_PTCFG 2
+	param set-default UXRCE_DDS_AG_IP 170461697
+	param set-default UXRCE_DDS_CFG 1000
+else
+	# Mavlink ethernet (CFG 1000)
+	param set-default MAV_2_CONFIG 1000
```

Comment:
Shouldn't we disable this by default? The resource use with these defaults together with multi EKF and UAVCAN screwed us over in a test yesterday (on main from November). I would rather document how to set things up instead of risking people running out of RAM and/or CPU because of things they might not even use.

Maybe I'm missing the broader context, it's just a suggestion from my perspective.